boost/meta.yaml

```diff
@@ -17,6 +17,8 @@ requirements:
     - libpython         [win]
```

Comment:
Ah, sorry. I haven't noticed it. I think it wasn't there before, so I don't know why is not reflected on the diff :-)